tests/sdk/test_traversal.py

```diff
@@ -466,29 +445,18 @@ def test_workflow_without_data_aggregation(self):
         wf = constant_return.model
         assert wf.data_aggregation is None
 
+    @pytest.mark.filterwarnings("ignore:data_aggregation")
```

Comment:
This test will be removed with removal of data_aggregation from the API. We dont have to add TODO - when data_aggregation parameter will be gone, this test will stop passing as a result so it would be removed as part of that PR.